docs/source/api/python/dgl.graphbolt.rst

```diff
@@ -56,7 +56,7 @@ Standard Implementations
 
     OnDiskDataset
     BuiltinDataset
-    CSCSamplingGraph
+    FusedCSCSamplingGraph
```

Comment:
will we add a new wrapper or interface class such as `CSCSamplingGraph` to incorporate fused and non-fused classes?

Reply:
non-fused one will be named as `CSCSamplingGraph`